src/hotspot/share/classfile/bytecodeAssembler.hpp

```diff
@@ -93,9 +93,9 @@ class BytecodeCPEntry {
     return bcpe;
   }
 
-  static BytecodeCPEntry string(u2 index) {
+  static BytecodeCPEntry string(Symbol* symbol) {
     BytecodeCPEntry bcpe(STRING);
-    bcpe._u.string = index;
+    bcpe._u.utf8 = symbol;
```

Comment:
It seems to me that these changes have killed off the use of the `_u.string` union component.

I also can't make a connection between these changes and the duplicate removals.

Reply:
I added some comments and renamed a couple things to make it make more sense, and remove u.string component since you're right, it's no longer used.